packages/manager/modules/sms/src/sms/sms/compose/telecom-sms-sms-compose.html

```diff
@@ -142,6 +142,21 @@ <h1 data-translate="sms_sms_compose_title"></h1>
                             </div>
                         </div>
 
+                        <div class="form-group" data-ng-hide="$ctrl.isVirtualNumber()">
```

Comment:
Maybe ? In order to generate item when needed
```suggestion
                        <div class="form-group" data-ng-if="!$ctrl.isVirtualNumber()">
```